test/studies/amr/advection/level/LevelBC_AdvectionCTU.chpl

```diff
@@ -13,9 +13,9 @@ class ZeroInflowBC: LevelBC {
 
     for grid in level.grids {
 
-      for ghost_domain in level.boundary(grid) {
+      for ghost_domain in level.boundary(grid)! {
```

Comment:
What does this `!` on the iterator expression do?

Reply:
It looks like it promotes a nilable element to a non-nilable, but I'm not sure if that's intentional or not (?).

Reply:
It does work, though.